.changeset/lazy-items-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/ember-toucan-core': minor
```

Comment:
A `patch` here is fine since we are `pre-1.0`.  Minor versions are reserved for breaking changes!